src/Config/Azure.DataApiBuilder.Config.csproj

```diff
@@ -23,4 +23,8 @@
       <PackageReference Include="Humanizer" />
   </ItemGroup>
 
+    <ItemGroup>
+        <ProjectReference Include="..\Product\Azure.DataApiBuilder.Product.csproj" />
```

Comment:
Was Product proj added since adding a dependency on Service would lead to circular reference?

Reply:
yes